packages/nouns-contracts/contracts/StreamEscrow.sol

```diff
@@ -0,0 +1,311 @@
+// SPDX-License-Identifier: GPL-3.0
+
+/// @title Stream Escrow
+
+/*********************************
+ * ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░ *
+ * ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░ *
+ * ░░░░░░█████████░░█████████░░░ *
+ * ░░░░░░██░░░████░░██░░░████░░░ *
+ * ░░██████░░░████████░░░████░░░ *
+ * ░░██░░██░░░████░░██░░░████░░░ *
+ * ░░██░░██░░░████░░██░░░████░░░ *
+ * ░░░░░░█████████░░█████████░░░ *
+ * ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░ *
+ * ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░ *
+ *********************************/
+
+pragma solidity ^0.8.19;
+
+import { IStreamEscrow } from './interfaces/IStreamEscrow.sol';
+import { INounsToken } from './interfaces/INounsToken.sol';
+
+contract StreamEscrow is IStreamEscrow {
```

Comment:
just in case, might be a good idea to have a "rescueTokens" function that allows the DAO to withdraw and ERC20s that might get sent here. I know we don't intend for erc20s to be sent here, but we have seen many accidents onchain, and this feels like an easy mitigation. wdyt?

Reply:
done d82688b0a3ae7060c21278be948a5ebdc3aee7a1